Form-Controls/styles.css

```diff
@@ -0,0 +1,31 @@
+body {
+  font-family: "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
+  color: black;
+  font-size: 30px;
+  margin: 2rem;
+  background-color:blanchedalmond;
+}
+
+.select-colour {
+  font-size: 1.5 rem;
+  padding: 1rem;
+  margin: 1rem;
+}
+
+.select-size {
+  font-size: 1.5 rem;
+  padding: 1rem;
+  margin: 1rem;
+}
+
+.validate-account {
+  font-size: 1.5 rem;
+  padding: 1rem;
+  margin: 1rem;
+}
+
+.select-delivery-date {
```

Comment:
It looks like you've applied the same styles to all these classes. Is there a way you could apply one style to all the classes? What does "class" actually mean?